app/javascript/dashboard/routes/dashboard/settings/profile/Wrapper.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div
+    class="flex flex-col justify-between flex-1 h-full m-0 overflow-auto bg-white dark:bg-ash-900"
```

Comment:
@muhsin-k `dark:bg-ash-900` not working, let us use `bg-slate-900`
```suggestion
    class="flex flex-col justify-between flex-1 h-full m-0 overflow-auto bg-white dark:bg-slate-900"
```